lavalink/WebSocket.py

```diff
@@ -20,6 +20,8 @@ def __init__(self, lavalink, host, password, ws_port, ws_retry, shard_count):
         self._uri = 'ws://{}:{}'.format(self._host, self._port)
         self._shards = shard_count
 
+        self._shutdown = asyncio.Event()

```

Comment:
Perhaps it's worth using a boolean for this, `self._is_shutdown` - I generally try to refrain from interacting with asyncio Events/Futures etc